cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -833,6 +857,10 @@ func hcsPropertiesToWindowsStats(props *hcsschema.Properties) *stats.Statistics_
 
 func (ht *hcsTask) Stats(ctx context.Context) (*stats.Statistics, error) {
 	s := &stats.Statistics{}
+	if ht.jobContainer {
+		return s, nil
```

Comment:
does this mean there are no stats for a job container?

Reply:
There are now, this needs to be updated: https://github.com/microsoft/hcsshim/commit/012856b73109e9eccbff67429a85efec49121238#diff-5271eb15b04ec6e383775402a2fc4eebe92d7f86a587a6f64fd20acdb88a35b2